src/components/View.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { IViewProps } from '../models/propsModels'
+import './View.css'
+
+const View = ({ view, changeView }: IViewProps) => {
+    const changeViewOnClick = (btn: string): void => {
```

Comment:
In that case you can simplify thing by passing down not a toggle function but plain `setIsNewCustomerView` function. That change allow you remove this handler function completely and use plain `setState` fn in buttons.

 ```tsx
return (
    <div className="view-btns">
        <button
            className={'view-btn btn ' + (view ? 'not_active' : 'active')}
            onClick={(): void => setIsNewCustomerView(false)}
        >
            All Promotions
        </button>
        <button
            className={'view-btn btn ' + (view ? 'active' : 'not_active')}
            onClick={(): void => setIsNewCustomerView(true)}
        >
            New Customers
        </button>
    </div>
)
```